option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message HeroCheerRequest
{
 required U64Type curGuid = 1;
 required uint32 slot = 2;
}
